parsers/BORNHOLM_POWERLAB.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+# The arrow library is used to handle datetimes
+from datetime import datetime
+from logging import Logger, getLogger
+from typing import List, Optional
+
+from pytz import timezone
+from requests import Session
+
+from electricitymap.contrib.lib.models.event_lists import (
+    ExchangeList,
+    ProductionBreakdownList,
+)
+from electricitymap.contrib.lib.models.events import ProductionMix
+from electricitymap.contrib.lib.types import ZoneKey
+
+PRODUCTION_MAPPING = {
+    "wind": "wind_turbines",
+    "biomass": "factory",
+    "solar": "solar_cells",
+}
+LATEST_DATA_URL = "http://bornholm.powerlab.dk/visualizer/latestdata"
+
+SOURCE = "bornholm.powerlab.dk"
+TIMEZONE = timezone("Europe/Copenhagen")
+
+
+def fetch_production(
+    zone_key: ZoneKey = ZoneKey("DK-BHM"),
+    session: Optional[Session] = None,
+    target_datetime: Optional[datetime] = None,
+    logger: Logger = getLogger(__name__),
+) -> List[dict]:
+    """Requests the last known production mix (in MW) of a given country."""
+    if target_datetime:
+        raise NotImplementedError("This parser is not yet able to parse past dates")
+    if session is None:
+        session = Session()
+    response = session.get(LATEST_DATA_URL).json()
+    production = ProductionBreakdownList(logger)
+    production.append(
+        zoneKey=zone_key,
+        datetime=datetime.fromtimestamp(response["latest"], tz=TIMEZONE),
+        source=SOURCE,
+        production=ProductionMix(
+            biomass=response["sub"]["factory"],
+            solar=response["sub"]["solar_cells"],
+            wind=response["sub"]["wind_turbines"],
+        ),
```

Comment:
granted these are only a single value and there are no math involved but I think we should still consider using add_value() for them.

Reply:
In this case I find it easier to have the modes in the constructor. No real reason behind this but it's more for convenience, since both are supported I don't think we should favor one over the other.